packages/components/src/theme/ontologybrowser.scss

```diff
@@ -1,4 +1,34 @@
+.concept-filter-view {
```

Comment:
not sure where this goes, but in quick manual testing, I noticed that putting the mouse over the "Find Concepts By Tree" and "Close Browser" text does not us the pointer cursor.

Reply:
done.
Side question should this block be moved to the platform/core App's `scss` file?

Reply:
Yes, if the class name `concept-filter-view` is only used in the core module entryPoint code, then I think moving it to that scss file would be good.